### DISCUSSION TITLE
Hardware testing

### DISCUSSION THREAD

**Discussion by elsirion** [fedimint/fedimint#1445] [General]
We should try to compile `fedimint` for ARM and try to run a federation on Raspberry Pis.

**Comment by justinmoon** [comment]
We successfully ran a 3/4 federation on Raspberry Pis last summer, but there hasn't been much testing since then. Would be good to revisit this in February!

**Comment by m1sterc001guy** [comment]
I have an old Raspberry Pi 3 (1GB memory) that would be good for testing low resource availability
